src/app/testing/gene-mocks.ts

```diff
@@ -3278,7 +3280,8 @@ export const geneMock3: Gene = {
   ],
   nominations: null,
   is_adi: false,
-  is_tep: true
+  is_tep: true,
+  resource_url: null
```

Comment:
Thanks for allowing the null type for `resource_url`! I'm still curious about the data in this mock though -- since `is_tep` is true, should `resource_url` contain a URL, given the data validation rules described in the ticket? Alternately, `is_tep` could be set to false and `resource_url` maintained as null. What do you think?

For reference, here is the data validation rule described in the ticket: 

> If the data has TRUE boolean(s) but no resource_url, that is a data issue. You shouldn’t see that, as I have validation rules in place to detect it.

Where the booleans referred to are: 

>   - gene_info.is_tep == TRUE
>   - gene_info.is_adi == TRUE

Reply:
I think since this is a scenario that shouldn't occur upstream, I was mostly concerned that it the value was null, the page would at least continue to load.  I focused on making sure the cards didn't show inadvertently.  Ideally, I would want to log and monitor for data issues in the future as well as incorporating some data quality checks, but I consider these items as a future enhancement.  Let me know if that makes sense!

Reply:
Ah okay, sounds good!